src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Please, make new Random() class level field
Do not forget `private` `final`

Reply:
why Random() class level field should be done and private final Color[ ] colors = Color.value(); not needed, in which documentation can I look this up to avoid mistakes in the future?